optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/SolverBuildTimeConfig.java

```diff
@@ -33,11 +33,12 @@
 public class SolverBuildTimeConfig {
 
     /**
-     * Enable runtime assertions to detect common bugs in your implementation during development.
-     * Defaults to {@link EnvironmentMode#REPRODUCIBLE}.
+     * Enable daemon mode. In daemon mode, non-early termination pauses the solver instead of stopping it,
+     * until the next problem fact change arrives. This is often useful for real-time planning.
+     * Defaults to "false".
      */
     @ConfigItem
-    Optional<EnvironmentMode> environmentMode;
+    Optional<Boolean> daemon;
```

Comment:
In SolverConfig.java, "daemon" is just after "EnviromnentMode". The XSD enforces this order in solverConfig.xml.
Let's keep that order for this file too, and the docs of quarkus properties?